File: ruby/pow/spec/pow_spec.rb
require 'spec_helper'
require_relative '../lib/pow'

describe '#pow' do 
  it 'returns something' do 
    expect(pow(2,3)).should_not be_nil
  end

  it 'returns the base raised to the exponent power' do
    expect(pow(2,3)).to eq(8)
  end
end

File: javascript/app_academy_test_1/pow/spec/pow_spec.js
// **Q1**: Write a method, `pow`, that takes two (non-negative, integer)
// numbers, `base` and `exponent` and returns `base` raised to the
// `exponent` power. (No fair using Ruby's `base ** exponent` notation!).

var pow = require('../lib/pow.js');

describe('pow', function() {
    it('is a function', function () {
        expect(typeof pow).toBe('function');
    });
});